### PR TITLE
chore: wearables max supply update

### PR DESCRIPTION
**Pull request by ghost**
## Description

Wearables max supply update.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

-   [X] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-   [ ] ~added `!` to the type prefix if API or client breaking change~
-   [ ] ~provided a link to the relevant issue or specification~
-   [X] included the necessary unit and integration tests
-   [X] code is adequately commented
-   [ ] ~updated the relevant documentation or specification~
-   [X] reviewed "Files changed" and left comments if necessary
-   [X] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

-   [X] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
-   [ ] ~confirmed `!` in the type prefix if API or client breaking change~
-   [ ] ~confirmed all author checklist items have been addressed~
-   [X] reviewed service design and naming
-   [ ] ~reviewed documentation is accurate~
-   [X] reviewed tests and test coverage
-   [X] manually tested (if applicable)

### CODE REVIEW

**Comment by ghost** [comment]
Ensure the following env vars are set:

```
DEPLOYER_PRIVATE_KEY
RPC_URL
```

The `DEPLOYER_PRIVATE_KEY` can be set to any key (if you require an RPC endpoint for mainnet, please tag me).

To then generate the calldata, run:

```
npm run deploy:mainnet 
```

Output:


```
schedule batch calldata with  2  actions
  0x8f2a0bb000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000000ceed74372c218018988b0808777b9ee5cbe5ff0e8fdd6d581fe7bc98e690205a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf18000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf180000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004b00000000000000000000000000000000000000000000000000000000000001a4000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004c00000000000000000000000000000000000000000000000000000000000186a000000000000000000000000000000000000000000000000000000000

execute batch calldata
  0xe38335e500000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001600000000000000000000000000000000000000000000000000000000000000000ceed74372c218018988b0808777b9ee5cbe5ff0e8fdd6d581fe7bc98e690205a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf18000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf180000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004b00000000000000000000000000000000000000000000000000000000000001a4000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004c00000000000000000000000000000000000000000000000000000000000186a000000000000000000000000000000000000000000000000000000000
```

**Comment by KBryan** [comment]
LGTM

```
0x8f2a0bb000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000001800000000000000000000000000000000000000000000000000000000000000000ceed74372c218018988b0808777b9ee5cbe5ff0e8fdd6d581fe7bc98e690205a00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf18000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf180000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004b00000000000000000000000000000000000000000000000000000000000001a4000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004c00000000000000000000000000000000000000000000000000000000000186a000000000000000000000000000000000000000000000000000000000
```
  warping to 1712889803
  execute batch calldata
  ```
0xe38335e500000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000001600000000000000000000000000000000000000000000000000000000000000000ceed74372c218018988b0808777b9ee5cbe5ff0e8fdd6d581fe7bc98e690205a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf18000000000000000000000000792e36c772f6da6280fa43159792f89e7444cf180000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004b00000000000000000000000000000000000000000000000000000000000001a4000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000044ccb3cf07000000000000000000000000000000000000000000000000000000000000004c00000000000000000000000000000000000000000000000000000000000186a000000000000000000000000000000000000000000000000000000000
```

**Review by KBryan — APPROVED** [review]
lgtm

File: proposals/zips/zip009.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "maxSupply": 420,
```
Comment (ElliotFriedman) [inline proposals/zips/zip009.json:3]: nice

**Review by ElliotFriedman — COMMENTED** [review]
🚀🚀🚀

LGTM!

**Pull request merged**